File: capitulos/cap05-navigation-ok/objc/BemVindo-Modal/BemVindo/Classes/MeuSegundoViewController.m
//
//  MeuSegundoViewController.m
//  BemVindo
//
//  Created by Ricardo Lecheta on 7/9/14.
//  Copyright (c) 2014 Ricardo Lecheta. All rights reserved.
//

#import "MeuSegundoViewController.h"

@implementation MeuSegundoViewController

@synthesize msg;

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    // Copia a mensagem para o label
    label.text = msg;
    
    NSLog(@"%@", msg);
}


- (IBAction)voltar:(id)sender {
    [self dismissViewControllerAnimated:YES completion:nil];
}

@end

File: capitulos/cap05-navigation-ok/01-BemVindo-Modal/BemVindo/Classes/MeuSegundoViewController.swift
//
//  MeuSegundoViewController.swift
//  BemVindo
//
//  Created by Ricardo Lecheta on 28/04/18.
//  Copyright © 2018 Ricardo Lecheta. All rights reserved.
//

import UIKit

class MeuSegundoViewController: UIViewController {
    
    // Label desenhado xib xib
    @IBOutlet var label : UILabel!

    // Mensagem que será passada como parâmetro
    var msg = ""

    override func viewDidLoad() {
        super.viewDidLoad()

        // Copia o conteúdo da mensagem "msg" para o label
        label.text = msg
    }

    // Fecha este view controller
    @IBAction func voltar(_ sender: UIButton) {
        self.dismiss(animated: true)
    }
}
